src/telliot_feeds/reporters/tellorflex.py

```diff
@@ -362,7 +362,7 @@ async def report(self) -> None:
         """Submit latest values to the TellorFlex oracle."""
 
         while True:
-            online = self.is_online()
+            online = await self.is_online()
```

Comment:
are you sure you should add an `await` here?

Reply:
i believe so cause its an async function

Reply:
also don't forget `tox  -e style` :)

Reply:
didn't forget

Reply:
its unrelated to this, its being caused by something else

Reply:
oh right. i ran into that too

Reply:
src/telliot_feeds/sources/price_aggregator.py:21:1: B024 PriceAggregator is an abstract base class, but it has no abstract methods. Remember to use @abstractmethod, @abstractclassmethod and/or @abstractproperty decorators.